Chapter-3-Exercises/jjhampton-ch3-recursion.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+/* -------------------------------
+We’ve seen that % (the remainder operator) can be used to test whether a number is even or odd by using % 2 to check whether it’s divisible by two. Here’s another way to define whether a positive whole number is even or odd:
+
+* Zero is even.
+
+* One is odd.
+
+* For any other number N, its evenness is the same as N - 2.
+
+Define a recursive function isEven corresponding to this description. The function should accept a number parameter and return a Boolean.
+-------------------------------*/
+
+function isEven(testedNumber) {
+    // using Math.abs() to conditionally check against only absolute aka positive values of numbers - this  prevents overflowing call stack and allows recursion stop condition to eventually get called
+    if (Math.abs(testedNumber) === 0)  
```

Comment:
Yeah - it was interesting to see how you accomplished the same thing w/o calling methods on the `Math` object.  Always cool to see different approaches to solving a problem.